src/CurrencyNetwork.ts

```diff
@@ -10,21 +11,21 @@ export class CurrencyNetwork {
   }
 
   public getInfo (networkAddress: string): Promise<any> {
-    if (!this.utils.checkAddress(networkAddress)) {
+    if (!ethUtils.isValidChecksumAddress(networkAddress)) {
```

Comment:
And please use the new created checkAddress function also here in the code, so it is the same everywhere